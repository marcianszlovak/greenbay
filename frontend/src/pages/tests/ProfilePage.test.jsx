import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router-dom';

import ProfilePage from '../ProfilePage';
import store from '../../store';

describe('ProfilePage Component', () => {
  const MockProfilePage = (
    <MemoryRouter>
      <Provider store={store}>
        <ProfilePage />
      </Provider>
    </MemoryRouter>
  );

  it('should match the snapshot', () => {
    const tree = renderer.create(MockProfilePage).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
