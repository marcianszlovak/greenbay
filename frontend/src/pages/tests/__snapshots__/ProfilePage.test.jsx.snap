// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfilePage Component should match the snapshot 1`] = `
<div
  className="row"
>
  <div
    className="col-md-3"
  >
    <h2>
      User Profile
    </h2>
    <form
      className=""
      onSubmit={[Function]}
    >
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="name"
        >
          Name
        </label>
        <input
          className="form-control"
          id="name"
          onChange={[Function]}
          placeholder="Enter name"
          type="name"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="email"
        >
          Email Address
        </label>
        <input
          className="form-control"
          id="email"
          onChange={[Function]}
          placeholder="Enter email"
          type="email"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="money"
        >
          Money
        </label>
        <input
          className="form-control"
          id="money"
          onChange={[Function]}
          placeholder="Enter money amount"
          type="number"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="password"
        >
          Password
        </label>
        <input
          className="form-control"
          id="password"
          onChange={[Function]}
          placeholder="Enter password"
          type="password"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="confirmPassword"
        >
          Confirm Password
        </label>
        <input
          className="form-control"
          id="confirmPassword"
          onChange={[Function]}
          placeholder="Confirm password"
          type="password"
          value=""
        />
      </div>
      <button
        className="btn btn-primary"
        disabled={false}
        type="submit"
      >
        Update
      </button>
    </form>
  </div>
  <div
    className="col-md-9"
  >
    <h2>
      My Orders
    </h2>
    <div
      className="table-responsive"
    >
      <table
        className="table-sm table table-striped table-bordered table-hover"
      >
        <thead>
          <tr>
            <th>
              ID
            </th>
            <th>
              DATE
            </th>
            <th>
              TOTAL
            </th>
            <th>
              PAID
            </th>
            <th>
              DELIVERED
            </th>
            <th />
          </tr>
        </thead>
        <tbody />
      </table>
    </div>
  </div>
</div>
`;
